src/Npgsql/Internal/NpgsqlConnector.cs

```diff
@@ -2361,6 +2355,13 @@ internal async Task ExecuteInternalCommand(byte[] data, bool async, Cancellation
 
         #region Misc
 
+        /// <summary>
+        /// Creates and returns a <see cref="NpgsqlCommand"/> object associated with the <see cref="NpgsqlConnector"/>.
+        /// </summary>
+        /// <param name="cmdText">The text of the query.</param>
+        /// <returns>A <see cref="NpgsqlCommand"/> object.</returns>
+        public NpgsqlCommand CreateCommand(string? cmdText = null) => new(cmdText, this);
```

Comment:
Sorry, my bad - I think you were right the first time. In terms of Intellisense and discoverability, it's better for this to be "hidden" on NpgsqlConnector rather than very exposed as a constructor on NpgsqlCommand which people may use be accident... I guess that was your thinking in the first place.

Reply:
My thinking was to not change the existing API whenever possible, that's pretty much it :)

Reply:
Basically we should try hard not to make users accidentally bring in Internal types, so we should avoid exposing anything Internal publicly on non-Internal types...